tgui/packages/tgui/interfaces/SpecMenu.js

```diff
@@ -0,0 +1,99 @@
+import { useBackend } from "../backend";
+import { Box, Button, Flex, Section, Divider } from "../components";
+import { Window } from "../layouts";
+
+export const SpecMenu = (props, context) => {
+  return (
+    <Window resizable theme="nologo">
+      <Window.Content>
+        <Flex justify="space-around">
+          <HemoMenu />
+          <Divider vertical={1} />
+          <UmbrMenu />
+          <Divider vertical={1} />
+          <GarMenu />
+        </Flex>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const HemoMenu = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    subclasses,
+  } = data;
+  return (
+    <Flex.Item grow={1} basis="33%">
+      <Section title="Hemomancer">
+        <p>Focuses on blood magic and the manipulation of blood around you.</p>
+        <p><b>Vampiric claws</b>: unlocked at 150 blood, allows you to summon a robust pair of claws that attack rapidly and drain a targets blood.</p>
+        <p><b>Blood tendrils</b>: unlocked at 250 blood, allows you to slow everyone in a targeted 3x3 area after a short delay.</p>
+        <p><b>Sanguine pool</b>: unlocked at 400 blood, allows you to travel at high speeds for a short duration. Doing this leaves behind blood splatters. You can move through anything but walls and space when doing this.</p>
+        <p><b>Blood eruption</b>: unlocked at 600 blood, allows you to manipulate all nearby blood splatters into spikes that impale anyone stood ontop of them.</p>
+        <p><b>Full power</b>
+          <Divider />
+          <b>The blood bringers rite</b>: when toggled you will rapidly drain the blood of everyone who is nearby and use it to heal yourself and remove stamina damage.
```

Comment:
**(This review and below are just opinion.)**

Uppercase looks better than lowercase for the descriptions, I think:
![image](https://user-images.githubusercontent.com/57483089/132144300-169aef8b-ca0d-4048-8ee5-37cf918df504.png) ![image](https://user-images.githubusercontent.com/57483089/132144285-c85f70e5-d991-4698-a235-731b66a6cd46.png)
```suggestion
        <p><b>Vampiric claws</b>: Unlocked at 150 blood, allows you to summon a robust pair of claws that attack rapidly and drain a targets blood.</p>
        <p><b>Blood tendrils</b>: Unlocked at 250 blood, allows you to slow everyone in a targeted 3x3 area after a short delay.</p>
        <p><b>Sanguine pool</b>: Unlocked at 400 blood, allows you to travel at high speeds for a short duration. Doing this leaves behind blood splatters. You can move through anything but walls and space when doing this.</p>
        <p><b>Blood eruption</b>: Unlocked at 600 blood, allows you to manipulate all nearby blood splatters into spikes that impale anyone stood ontop of them.</p>
        <p><b>Full power</b>
          <Divider />
          <b>The blood bringers rite</b>: When toggled you will rapidly drain the blood of everyone who is nearby and use it to heal yourself and remove stamina damage.
```